commands/ServerEvents.cs

```diff
@@ -440,6 +440,9 @@ public void Help(IUser admin)
 
             admin.Print("/id");
             admin.Print("/info");
+            admin.Print("/report <report>");
+            admin.Print("/reports");
+            admin.Print("/clearreports");

```

Comment:
Ok. View the changes: https://github.com/Nemo2478/sb0t-1/commit/e0c3fb8bfec8968a5d95b96f25eec82635363958